src/olympia/addons/models.py

```diff
@@ -352,9 +352,6 @@ class Addon(OnChangeMixin, ModelBase):
 
     whiteboard = models.TextField(blank=True)
 
-    # Whether the add-on is listed on AMO or not.
-    is_listed = models.BooleanField(default=True, db_index=True)
```

Comment:
We should file an issue to get rid of the column asynchronously. I've checked and dev/stage (presumably prod too, but that needs to be verified before merging) do have a in-database default for this field, so it's fine to remove from the models and do the migration later.